upstream/jobs.go

```diff
@@ -263,9 +269,12 @@ func reconcileTable(ctx context.Context, config UpstreamConfig, table pushableTa
 			})
 			failedItems := lo.Filter(items, func(item models.DBTable, _ int) bool {
 				_, ok := failedOnes[item.PK()]
+				if ok {
+					fkFailed.IDs = append(fkFailed.IDs, item.PK())
```

Comment:
Can we limit these to first 10 only

Reply:
Added that logic in its json Marshal function